src/ProgressVS_2017/ProgressVS.csproj

```diff
@@ -22,7 +22,7 @@
     <DefineConstants>DEBUG;TRACE</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
-    <CodeAnalysisRuleSet>$(ReferencedDirectory)\ProgressVS.ruleset</CodeAnalysisRuleSet>
+    <CodeAnalysisRuleSet>ProgressVS.ruleset</CodeAnalysisRuleSet>
```

Comment:
Why are you changing this path? We were using a local defined variable (around the top of the file) defining where to look for (i.e. the VS2015 equivalent of the current project).